paddle/fluid/framework/ir/mkldnn/matmul_activation_mkldnn_fuse_pass.cc

```diff
@@ -77,6 +77,16 @@ void MatmulActivationMkldnnFusePass::FuseMatmulAct(
               ? "gelu_tanh"
               : "gelu_erf";
     }
+
+    if (matmul_type == "matmul") {
+      matmul_op->SetType("matmul_v2");
+      matmul_op->SetAttr("trans_x", matmul_op->GetAttr("transpose_X"));
+      matmul_op->SetAttr("trans_y", matmul_op->GetAttr("transpose_Y"));
+      auto matmul_alpha = matmul_op->GetAttrIfExists<float>("alpha");
+      if (matmul_alpha != 1.0f) {
+        matmul_op->SetAttr("alpha", matmul_alpha);
+      }
```

Comment:
`alpha` is not a attribute of `matmul_v2`, I'm not sure whether this `alpha` can work in kernel of `matmul_v2`, please do a check ~

Reply:
Yes, it works because `alpha` is added as extra OneDNN-specific attribute in _ops_extra_info.h_. I'm aware that it's not elegant solution, so in next PR I will introduce fused_matmul with all necessary attributes declared in OPMaker.